Framework/MDAlgorithms/test/ConvertToQ3DdETest.h

```diff
@@ -407,16 +389,13 @@ class ConvertToQ3DdETest : public CxxTest::TestSuite {
   void xtestTransf_GL() {
```

Comment:
Are all of these tests disabled (prefixed with `x`)? Do they not pass?

(I know this is not relevant to the work here, just worth mentioning)

Reply:
Good catch! I was so focused on the changes I hadn't spotted the `x`. I'd love to know why we have disabled tests here. I'll try and find out why.